src/Sentry.Maui/Internal/Extensions.cs

```diff
@@ -53,21 +53,26 @@ public static void AddElementInfo(this IDictionary<string, string> data,
         // The element ID seems to be mostly useless noise
         //data.Add(prefix + nameof(element.Id), element.Id.ToString());
 
-        if (element.StyleId != null)
+        if (element.AutomationId is { } automationId)
```

Comment:
question @jamescrosswell

Do we need an opt-in Option for this as well?

See
- `SentryMauiOptions.IncludeTextInBreadcrumbs`
- `SentryMauiOptions.IncludeTitleInBreadcrumbs`

Do I understand it right that we DO NOT need an opt-in Option for the `AutomationId`,
because it won't include PII,
whereas `Text` and `Title` may contain PII and therefore need the explicit opt-in?

Reply:
That's my understanding: the `AutomationId` it's a developer/QA-friendly identifier used mainly by QA automation software, so I highly doubt it would contain PII.

https://learn.microsoft.com/en-us/dotnet/api/microsoft.maui.controls.element.automationid?view=net-maui-9.0

Reply:
Doesn't sound like a PII risk then 👍🏻